File: Data Structures & Algorithms/longest-common-prefix/submission-1.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        string ans = "";
        for(int j=0; j<strs[0].size();j++){
            char current = strs[0][j];

            for(int i=0; i<strs.size();i++){
                if(j>strs[i].size() || current != strs[i][j]){
                    return ans;
                }
            }
            ans += current;

        }
        return ans;
    }
};

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-1.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        cur = root
        q = deque()
        q.append(cur)
        ans = []
        while q:
            l = []
            qlen = len(q)
            for i in range(qlen):
                node = q.popleft()
                if node:
                    l.append(node.val)
                    # This is what is taking my traversal to next level
                    q.append(node.left)
                    q.append(node.right)
            if l:
                ans.append(l)

        return ans
                
